ArduCopter/crash_check.cpp

```diff
@@ -294,8 +293,7 @@ void Copter::parachute_check()
     parachute.check_sink_rate();
 
     // check for angle error over 30 degrees
-    const float angle_error = attitude_control->get_att_error_angle_deg();
-    if (angle_error <= PARACHUTE_CHECK_ANGLE_DEVIATION_DEG) {
+    if (attitude_control->get_att_error_angle_deg() <= CRASH_CHECK_ANGLE_DEVIATION_DEG) {
```

Comment:
This doesn't look right...

```suggestion
    if (attitude_control->get_att_error_angle_deg() <= PARACHUTE_CHECK_ANGLE_DEVIATION_DEG) {
```